protocol/release_management.md

```diff
@@ -22,46 +20,70 @@ git checkout develop
    branch name.
 ```
 git flow release start 1.2.3
+# or
+git checkout -b release/1.2.3
 ```
 5. For projects where there is an application version file, update to match the
    release version and commit the file to the release branch.
 6. Publish the local release branch to Github
 ```
 git flow release publish 1.2.3
+# or
+git push origin release/1.2.3
 ```
-7. Move the staging tag to point to the release branch and push tag to Github
+7. _Deprecated(do not use in new apps)_ Move the staging tag to point to the release branch and push tag to Github
 ```
 git tag -a -f staging -m "release 1.2.3"
 git push -f --tags
 ```
-8. Bamboo(or Circle) CI will detect the release/1.2.3 branch and will run test suite and
-   static analysis tooling via Code Climate. If successful, application will be
+8. CircleCI or Jenkins will detect the `release/1.2.3` branch and will run test suite and
+   static analysis tooling. If successful, application will be
 deployed to the test environment. If not, Release Manager is responsible for
 fixing reported errors.
-9. Release Manager will assign the Deploy ticket to the Product Owner for
-   review.
+9. Release Manager will assign the Deploy issue to the Product Owner for
+   review, highlighting areas that need to be tested.
 10. If problems are detected:
-  1. The Development Team will make needed code changes on the release branch
-  2. Commit and push changes to the remote branch
-  3. Write additional tests, if necessary, to cover the problem space detected
-  4. Run application test suite and Code Climate to verify all tests and
+  - The Development Team will make needed code changes on the release branch
+  - Commit and push changes to the remote branch
+  - Write additional tests, if necessary, to cover the problem space detected
+  - Run application test suite and Code Climate to verify all tests and
      analysis tools pass.
 11. When Product Owner signs off on release, they will assign to Release
     Manager.
 
 ## Production
-1. When Product Owner has assigned the Deploy ticket back to the Release
+1. When Product Owner has assigned the Deploy issue back to the Release
    Manager, the release branch must be finished prior to deployment.
 ```
 git fetch master
 git flow release finish 1.2.3
 git push
 git push --tags
 ```
-2. Release Manager assigns to Operations for Production release.
-3. When application is released to Production Operations assigns Deploy ticket
+**or**
+```
+# update develop branch if changes were made in release branch
+git checkout develop
+git rebase release/1.2.3
+
```

Comment:
@mcritchlow  Should it be pushed to github `git push origin develop`?

Reply:
ah yes! good point. I'll update that now, thanks :)

Reply:
updated in ab3b7ce thanks for catching that @hweng 👍